exit_db/migrations/2018-07-30-160806_remove_unused/up.sql

```diff
@@ -0,0 +1,8 @@
+BEGIN TRANSACTION;
+CREATE TEMPORARY TABLE clients_backup(mesh_ip, wg_pubkey, wg_port, internal_ip, email, country);
```

Comment:
You're ommitting several fields here, take a look at the previous migration, the ones you're adding here are not the complete list minus {zip,luci_pass}